.github/workflows/test.yml

```
@@ -5,7 +5,11 @@ on:
   push:
     branches:
       - master
+    paths:
+      - '!container-images/**'
```

```suggestion
    paths-ignore:
      - 'container-images/**'
```
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

the bang-syntax is only to remove previously included paths, not to ignore whole structures